cookbooks/aws-parallelcluster-slurm/recipes/config_slurm_accounting.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: config_head_node
+#
+# Copyright:: 2013-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+file "#{node['cluster']['slurm']['install_dir']}/etc/slurm_parallelcluster_slurmdbd.conf" do
+  owner "#{node['cluster']['slurm']['dbduser']}"
+  group "#{node['cluster']['slurm']['dbdgroup']}"
+  mode '0600'
+end
+
+template "#{node['cluster']['scripts_dir']}/slurm/update_slurm_database_password.sh" do
+  source 'slurm/head_node/update_slurm_database_password.sh.erb'
+  owner 'root'
+  group 'root'
+  mode '0700'
+  variables(
+    secret_arn: node['cluster']['slurm']['database']['password_secret_arn'],
+    region: node['cluster']['region']
+  )
+  sensitive true
+end
+
+execute "update Slurm database password" do
+  user 'root'
+  group 'root'
+  command "#{node['cluster']['scripts_dir']}/slurm/update_slurm_database_password.sh"
+end
+
+service "slurmdbd" do
+  supports restart: false
```

Comment:
why this is false?

Reply:
I copied what is being done for the slurmctld in the `config_head_node.rb` recipe, and it's been here since the beginning.

I assume it's to trick Chef into full stopping and then starting the service without attempting a restart. I'm not sure why we would need this, but I guess here it doesn't make too much of a difference, since it's the first time we start the service (and maybe that's why we have `false` in the first place?).